content/en/docs/components/pipelines/v2/run-a-pipeline.md

```diff
@@ -1,80 +1,107 @@
-  alt="Create run button"
-  class="mt-3 mb-3 border border-info rounded">
+1. Click the **Runs** tab and then click **+ Create run**.
 
-5. Choose the pipeline you uploaded, provide a name, any run parameters, and click "Start".
-<img src="/docs/images/pipelines/start-a-run.png" 
-  alt="Start a run screen"
-  class="mt-3 mb-3 border border-info rounded">
+    <img src="/docs/images/pipelines/create-run.png" alt="Create run button" class="mt-3 mb-3 border border-info rounded">
 
+1. In the **Pipeline** field, click **Choose** to select the pipeline you uploaded.
 
-## 2. Run from the KFP SDK client
-You may also programatically submit pipeline runs from the KFP SDK client. The client supports two ways of submitting runs: from IR YAML or from a Python pipeline function. For either approach, start by instantiating a `Client` using the `host` URL of your KFP instance:
+1. Specify any additional run parameters and click **Start**.
 
-```python
-from kfp.client import Client
-client = Client(host='<YOUR_HOST_URL>')
-```
+    <img src="/docs/images/pipelines/start-a-run.png" alt="Start a run screen" class="mt-3 mb-3 border border-info rounded">
 
-To submit IR YAML for execution use the `.create_run_from_pipeline_package` method:
+## Run a pipeline from the KFP SDK client <a id="run_from_sdk_client"></a>
 
-```python
-client.create_run_from_pipeline_package('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
-```
+You can programmatically submit a pipeline run from the KFP SDK client in two ways:
 
-To submit a Python pipeline function for execution use the `.create_run_from_pipeline_func` convenience method, which wraps compilation and run submission into one method:
+* Submit an IR YAML
+* Submit a Python pipeline function for execution
 
-```python
-client.create_run_from_pipeline_func('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
-```
+To submit an IR YAML for execution:
+
+1. Instantiate a `Client` by specifying the `host` URL of your KFP instance:
+    ```python
+    from kfp.client import Client
+    client = Client(host='<YOUR_HOST_URL>')
+    ```
+    For more information about the `Client` constructor and method parameters, see the  [kfp.client][kfp-sdk-api-ref-client] in the [KFP SDK API reference][kfp-sdk-api-ref].
+
+1. Use the `.create_run_from_pipeline_package` method:
+
+    ```python
+    client.create_run_from_pipeline_package('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
+    ```
+
+To submit a Python pipeline function for execution:
+
+1. Instantiate a `Client` by specifying the `host` URL of your KFP instance:
+    ```python
+    from kfp.client import Client
+    client = Client(host='<YOUR_HOST_URL>')
+    ```
+
+1. Use the `.create_run_from_pipeline_func` convenience method. This wraps the compilation and run submission into one method.
+
+    ```python
+    client.create_run_from_pipeline_func('pipeline.yaml', arguments={'param': 'a', 'other_param': 2})
```

Comment:
not your work: Just noticed `.create_run_from_pipeline_func('pipeline.yaml'...)` should be `.create_run_from_pipeline_func(my_pipeline)`. This was my mistake!

While you're making other changes, could you update this please?

Reply:
Fix applied. Also removed the arguments.